src/main/api/2/studio-api-2.yaml

```diff
@@ -6920,6 +6920,307 @@ paths:
+        '201':
+          description: CREATED
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    patch:
+      tags:
+        - workflow
+      summary: Update workflow package
+      description: Required permission "requst_publish, publish"
+      operationId: workflowUpdatePackage
+      requestBody:
+        description: Workflow package entity to update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/WorkflowPackage'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  package:
+                    $ref: '#/components/schemas/WorkflowPackage'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/packages/{id}:
+    get:
+      tags:
+        - workflow
+      summary: Get workflow package by id
+      description: Required permission "requst_publish, publish"
+      operationId: workflowGetPackage
+      parameters:
+        - name: id
+          in: path
+          description: The workflow package ID
+          required: true
+          schema:
+            type: string
+            format: UUID                
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  package:
+                    $ref: '#/components/schemas/WorkflowPackage'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/packages/approve:
```

Comment:
If approving one package at a time, then this should be `/package/`

Reply:
Right now it works with a single package. But I think we should add support for multiple packages in the future

Reply:
Let's make it bulk.